### ISSUE TITLE
Error installing rancher to ACK using helm

### ISSUE THREAD

**Issue by sunny0826** [rancher/rancher#30972]
**What kind of request is this (question/bug/enhancement/feature request):**

bug

**Steps to reproduce (least amount of steps as possible):**

Installing rancher to ACK using helm

```bash
❯ helm install rancher rancher-stable/rancher \
 --namespace cattle-system \
 --set hostname=rancher.my.org
Error: chart requires kubeVersion: < 1.20.0 which is incompatible with Kubernetes v1.18.8-aliyun.1
```

**Result:**

**Other details that may be helpful:**

kubernetes version：v1.18.8-aliyun.1

**Environment information**
- Rancher version (`rancher/rancher`/`rancher/server` image tag or shown bottom left in the UI):
- Installation option (single install/HA):

**Cluster information**
- Cluster type (Hosted/Infrastructure Provider/Custom/Imported):
- Machine type (cloud/VM/metal) and specifications (CPU/memory):
- Kubernetes version (use `kubectl version`):

```
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.1", GitCommit:"c4d752765b3bbac2237bf87cf0b1c2e307844666", GitTreeState:"clean", BuildDate:"2020-12-19T08:38:20Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"18+", GitVersion:"v1.18.8-aliyun.1", GitCommit:"94f1dc8", GitTreeState:"", BuildDate:"2021-01-10T10:14:01Z", GoVersion:"go1.13.15", Compiler:"gc", Platform:"linux/amd64"}
```

- Docker version (use `docker version`):

```
(paste the output here)
```

**Comment by sowmyav27** [comment]
@sunny0826 Do you have the Rancher's latest released charts - v2.5.5 ? We have fixed this issue in rancher 2.5.5. Please do an `helm update` and install the chart, let us know if you still see an issue.

**Comment by sunny0826** [comment]
I pulled this chart and the information is as follows.

![image](https://user-images.githubusercontent.com/24563928/105787270-9d71c700-5fb9-11eb-95cd-439734fc1987.png)

**Comment by deniseschannon** [comment]
That's the older chart. Can you refresh your repro to get an updated chart? 

`Helm repo update` and then try again?

**Comment by sunny0826** [comment]
I've updated the helm repo, is this indeed the repo?

`rancher-stable`

**Comment by sowmyav27** [comment]
Fixes should be available in 2.5.5 from both `rancher-stable` and `rancher-latest` repo

**Comment by sunny0826** [comment]
I noticed that I am using the official repo of cn rancher, could you please check if this repo is up to date?
http://rancher-mirror.oss-cn-beijing.aliyuncs.com/server-charts/stable
